dnsutil/util.go

```
@@ -0,0 +1,91 @@
+package dnsutil
+
+import "strings"
+
+// AddDomain adds origin to s if s is not already a FQDN.
+func AddDomain(origin, s string) string {
+	// If you want the result to end in a ".", pass an origin that ends in a "."
```

Should we return an error on a non-qualified origin?